.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules/
 out/
 third_party/mongodb/
 third_party/node/
-.DS_Store
\ No newline at end of file
+.DS_Store
+package-lock.json
```

Comment:
Also, please sort this in alphabetical order.
BTW, do we really need this? I think this file will be created because you added child_process module to package.json.
If you don't add the module to package.json, I guess this might be unnecessary.

Reply:
You're right.
It's because I'm using over 5 version of npm.
I'll get rid of it.